libs/components/layout/src/lib/modules/box/box.component.html

```diff
@@ -9,8 +9,12 @@
   }"
 >
   <div class="sky-box-header-content">
```

Comment:
Instead of creating more DIV elements, could you simply add `justify-content: flex-end;` to the `.sky-box-header-content` CSS rule?

Reply:
Unfortunately, no. Adding `justify-content: flex-end;` would move both the header and the controls to the end of the flexbox.

Reply:
I just found a different way to accomplish the same thing. `margin-left: auto` on the controls selector. I'll make the change as soon as I can.

Reply:
https://jsfiddle.net/9b372nmL/18/
see here^